v02/mw/mw.txt

```diff
@@ -252533,7 +252533,7 @@ the latter married the two widows of his half-brother, and had by them <s1 slp1=
 <s>cirBawikA</s> ¦ <lex>f.</lex> <ab>id.</ab>, <ls n="Pañcat.">i, 3, 20/21.</ls><info lex="f"/>
 <LEND>
 <L>74351<pc>399,1<k1>cirBiwa<k2>cirBiwa<e>2
-<s>cirBiwa</s> ¦ <lex>nf.</lex> another kind of gourd, <ls>Car. i, 27, 101</ls> (<ab>v.l.</ab> <s>°Bawa</s>), <ls>Bhpr. v, 6, 36</ls> (<ab>cf.</ab> <s>indracirBiwI</s>; <s>kzudra-</s>, <s>kzetra-</s> and <s>gaja-cirBiwA</s>.)<info lex="f:n"/>
+<s>cirBiwa</s> ¦ <lex>n., °ṭā, f.</lex> another kind of gourd, <ls>Car. i, 27, 101</ls> (<ab>v.l.</ab> <s>°Bawa</s>), <ls>Bhpr. v, 6, 36</ls> (<ab>cf.</ab> <s>indracirBiwI</s>; <s>kzudra-</s>, <s>kzetra-</s> and <s>gaja-cirBiwA</s>.)<info lex="f:n"/>
```

Comment:
`<lex>n., °ṭā, f.</lex>` shoud be `<lex>n.</lex>, <s>°wA</s>, <lex>f.</lex>`
The idea is to have lexical information in `<lex>` tag and show the Sanskrit words inside `<s>` tag and in SLP1 transliteration.

Reply:
``` 
Instead of 
<lex>n., °ṭā, f.</lex>
better to use (IN MW )
<lex>n.</lex>, <s>°ṭA</s>, <lex>f.</lex>
```

Reply:
I am dubious about the re-writing of the lex fields.
It is very complicated to get all the details consistent with the assumptions of the displays (for tooltips, and for representing the feminine forms in `<s>A</s>` ).
For instance in 220377 (SUla), the suggested form of this pull request is
```
<lex>m.n.</lex> (<ab>ifc.</ab> 
 <lex type="hwifc">f A.</lex>). 
```
This has two problems
* missing period after f  (feminine)
* absence of \<s> tag around the A.
Could change to
```
<lex>m.n.</lex> (<ab>ifc.</ab> 
<lex type="hwifc">f. <s>A</s>. </lex>)
```

This displays properly with tooltips and transcoding of the A.

Here is the current cdsl form
```
<lex>mn.</lex> (<ab>ifc.</ab> 
<lex type="hwifc">f(<s>A</s>). </lex>)
```
This kind of difference in the lex tag markup between cdsl and the printed text probably occurs thousands of times in mw.txt.

**In my opinion,  the current cdsl form does NOT need to be changed.**  as it represents the salient information completely, despite being slightly different from the printed text.

However, If @drdhaval2785  and others think that all the lex tagging needs to be rewritten to be more consistent with the printed text form,  then this should be done by a program, so all the little quirky differences can be taken into account.
The individual submissions as in this pull request are not the way to go because:
* there are likely MANY thousands of changes
* the little details are hard to get right in manual changes

Better for @aumsanskrit to stick to the 'real errors',  hundreds of which he has found so far.

Reply:
Thank  you.  I completely understand.